app/test/unit/git/squash-test.ts

```diff
@@ -34,18 +45,269 @@ describe('git/cherry-pick', () => {
     expect(squashed.summary).toBe('Test Summary')
     expect(squashed.body).toBe('Test Body\n')
     expect(log.length).toBe(2)
+
+    // verify squashed commit contains changes from squashed commits
+    const squashedFiles = await getChangedFiles(repository, squashed.sha)
+    const squashedFilePaths = squashedFiles.map(f => f.path).join(' ')
+    expect(squashedFilePaths.includes('first.md')).toBeTrue()
+    expect(squashedFilePaths.includes('second.md')).toBeTrue()
```

Comment:
I think you can use the more concise `toContain` here:
```suggestion
    expect(squashedFilePaths).toContain('first.md')
    expect(squashedFilePaths).toContain('second.md')
```

Reply:
:D - I thought there should be something line this..